packages/atomic/src/utils/utils.ts

```diff
@@ -16,6 +16,33 @@ export function once<T extends unknown[]>(fn: (...args: T) => unknown) {
   };
 }
 
+export function listenOnce<K extends keyof HTMLElementEventMap>(
+  element: HTMLElement,
+  type: K,
+  listener: (this: HTMLElement, ev: HTMLElementEventMap[K]) => any,
+  options?: boolean | AddEventListenerOptions
+): void;
+export function listenOnce(
+  element: HTMLElement,
+  type: string,
+  listener: EventListenerOrEventListenerObject,
+  options?: boolean | AddEventListenerOptions
+): void;
```

Comment:
copied from `addEventListener`

Reply:
File is getting a lot of variety, maybe this should be moved to "event-utils"